cnames_active.js

```diff
@@ -1142,6 +1142,7 @@ var cnames_active = {
   "lgs": "lgsturkiye.github.io/lgs",
   "li": "mattsimmons1.github.io/li",
   "lifx": "fncxpro.github.io/lifx.js",
+  "lifeisyoung": "gru-lee.github.io",
```

Comment:
```suggestion
  "lifeisyoung": "lifeisyoung.github.io",
```
I'm guessing this should be targeting your new account name?